.travis.yml

```diff
@@ -68,13 +68,10 @@ addons:
 language: cpp
 compiler: clang
 
-before_install:
-    - curl --retry 10 --retry-max-time 120 -L "https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2" | tar xfj -
-
 install:
-  - export PATH=$PATH:$PWD/gcc-arm-none-eabi-5_4-2016q2/bin
+  - make arm_sdk_install
```

Comment:
Not sure this is the right thing to do, won't this just slow down the travis builds?

Apart from that: Interesting idea. :-)